app/src/types/shared.types.ts

```diff
@@ -92,3 +92,32 @@ export enum SortOrder {
   ASC = "asc",
   DESC = "desc",
 }
+
+export const comparators = ["=", "!=", "CONTAINS", "NOT_CONTAINS"] as const;
+
+export const filtersSchema = z.array(
+  z.object({
+    field: z.string(),
+    comparator: z.enum(comparators),
+    value: z.string(),
+  }),
+);
+
+export enum LoggedCallsFiltersDefaultFields {
```

Comment:
There's almost never a good reason to use a Typescript `enum` (in fact I believe I heard an interview with the TS architect who mentioned he regrets putting them in the language).

Union types are generally a better way to do the same thing. In this case I'd rewrite it as something like:

```ts
export const loggedCallsFiltersDefaultFields = ["Request", "Response", "Model", "Status Code"] as const;

export const LoggedCallsFiltersFieldTypes = (typeof loggedCallsFiltersDefaultFields)[number]
```